docs/controller.md

```diff
@@ -0,0 +1,93 @@
+## Controller - general information
+- Always up-to-date info with all the implementation details can be found in **src/controller.py**.
+- It is running on port 9001.
+- Requests must be in JSON structure, command are specified in `type` key, e.g. `{"type": "ping"}`.
+- All arguments (if needed) are being specified in the same root JSON, e.g. `{"type": "emulator-start", "version": "2.4.0"}`.
+- It is beneficial to always include `id` integer in the request, as it is being sent back in the response, and these two can be matched.
+- Response is always in JSON format, includes boolean `success` key, indicating if everything went as expected, e.g. `{"success": False, "error": "Unknown command - xyz"}`.
+- If the command does not ask anything, but simply wants to do something, the response will contain a confirmation in `response` key, e.g. `{"success": True, "response: "Emulator 2.4.0. started"}`.
```

Comment:
This is weird sentence. What do you mean "does not ask anything"?

Reply:
That they are not requesting any information, just telling `controller` to do something. Yes, it should be changed/removed, as apart from `ping` and `background-check`, all commands behave like this.

Reply:
"Response will always contain a confirmation of what happened in `response` key"?

Reply:
Yeah sounds better :+1: !

Reply:
`in the response key` :)